tests/spell.test.ts

```diff
@@ -20,4 +20,20 @@ describe('Test summoner spells fetching.', () => {
     const byName = await client.summonerSpells.findByName('Flash');
     expect(byName).toBe(flash);
   });
+
+  test('Check spell caching', async () => {
+    const cachedFlash = client.summonerSpells.cache.find((s) => s.id === 'SummonerFlash')!;
```

Comment:
You can simply do `client.summonerSpells.cache.get('SummonerFlash');` to get by ID instead of using `find`.